supervisor/jira_utils.py

```diff
@@ -293,14 +331,14 @@ def custom_enum_list(enum_class: Type[_E], name) -> list[_E] | None:
     if full:
         return FullIssue(
             **issue.__dict__,
-            description=issue_data["fields"]["description"],
+            description=issue_data["fields"]["description"] or "",
             comments=[
                 JiraComment(
                     authorName=c["author"]["displayName"],
                     authorEmail=c["author"].get("emailAddress"),
                     created=datetime.fromisoformat(c["created"]),
                     body=c["body"],
-                    id=c["id"],
+                    id=c["id"]
```

Comment:
Make sure you are using black for formatting, I believe it puts a trailing "," here